lib/pharos/host/debian/debian_stretch.rb

```diff
@@ -79,6 +74,14 @@ def reset
           CRIO_VERSION: CRIO_VERSION
         )
       end
+
+      def default_repositories
+        [Pharos::Configuration::Repository.new(
+          name: "pharos-kubernetes.list",
+          key: "https://bintray-pk.pharos.sh/?username=bintray",
+          contents: "deb https://dl.bintray.com/kontena/pharos-debian stretch main\n"
```

Comment:
if it needs linefeeds, maybe it should be an array?

```yaml
repositories:
  name: foofoo.list
  key: https://example.com
  contents:
    - deb https://dl.bintray.com/kontena/pharos-debian stretch main
    - deb https://dl.bintray.com/kontena/pharos-symbian sketch main
```

Reply:
It might or might not need newlines. This tries to be fully generic and the contents can vary heavily between distros